tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -19,8 +19,8 @@ commands:
       set -x
       RETURN_CODE=0
       PUBLIC_URL=$(oc get -n $NAMESPACE horizon horizon -o jsonpath='{.status.endpoint}')
-      STATUSCODE=$(oc exec -it deployment/horizon -- curl --silent --output /dev/stderr --head --write-out "%{http_code}" "$PUBLIC_URL/dashboard/auth/login/?next=/dashboard/" -k)
-      if test $STATUSCODE -ne 200; then
+      STATUSCODE=$(oc -n $NAMESPACE exec -it deployment/horizon -- curl --silent --output /dev/stderr --head --write-out "%{http_code}" "$PUBLIC_URL/dashboard/auth/login/?next=/dashboard/" -k)
```

Comment:
lgtm now. Let's see how kuttl likes it.

Reply:
looks good as job ran successfully , thank you